packages/orchestration/src/examples/basic-flows.contract.js

```diff
@@ -89,23 +22,18 @@ const makePortfolioAcctHandler = async (
  * @param {Zone} zone
  * @param {OrchestrationTools} tools
  */
-const contract = async (zcf, _privateArgs, zone, { orchestrate, vowTools }) => {
+const contract = async (
+  zcf,
+  _privateArgs,
+  zone,
+  { orchestrateAll, vowTools },
+) => {
   const makePortfolioHolder = preparePortfolioHolder(
     zone.subZone('portfolio'),
     vowTools,
   );
 
-  const makeOrchAccount = orchestrate(
-    'makeOrchAccount',
-    undefined,
-    makeOrchAccountHandler,
-  );
-
-  const makePortfolioAccount = orchestrate(
-    'makePortfolioAccount',
-    { makePortfolioHolder },
-    makePortfolioAcctHandler,
-  );
+  const orchFns = orchestrateAll(flows, { makePortfolioHolder });
```

Comment:
I vacillated on this but I don't see how providing `makePortfolioHolder` to both is a risk.

I think `orchestrateAll` is what we should steer people to in general because it gets the durable from a named export, which changing is known to break downstream, whereas the string argument was easily confused as a debugging name.

Reply:
👍 the second argument is convincing